src/bin/cratesfyi.rs

```diff
@@ -139,7 +139,26 @@ pub fn main() {
                     .short("p")
                     .long("priority")
                     .help("Priority of build (default: 5) (new crate builds get priority 0)")
-                    .takes_value(true))))
+                    .takes_value(true)))
+            .subcommand(SubCommand::with_name("priority")
+                .about("Interactions with build queue priorities")
+                .setting(AppSettings::ArgRequiredElseHelp)
+                .subcommand(SubCommand::with_name("set")
+                    .about("Set all crates matching the given pattern to a priority level")
+                    .arg(Arg::with_name("PATTERN")
+                        .index(1)
+                        .required(true)
+                        .help("See https://www.postgresql.org/docs/current/functions-matching.html"))
+                    .arg(Arg::with_name("PRIORITY")
+                        .index(2)
+                        .required(true)
+                        .help("The priority to give crates matching PATTERN"))
```

Comment:
Without this `remove` shows up as a subcommand of `add`:
```
$ cargo run --quiet queue priority set -h
cratesfyi-queue-priority-set 
Set all crates matching the given pattern to a priority level

...

SUBCOMMANDS:
    help      Prints this message or the help of the given subcommand(s)
    remove    Remove the prioritization of crates by the given pattern
```

```suggestion
                        .help("The priority to give crates matching PATTERN")))
```